forc/src/utils/mod.rs

```diff
@@ -1,8 +1,16 @@
 pub mod defaults;
 pub mod parameters;
 
+use anyhow::{anyhow, Result};
+use rustc_version::{version, Version};
+
 /// The `forc` crate version formatted with the `v` prefix. E.g. "v1.2.3".
 ///
 /// This git tag is used during `Manifest` construction to pin the version of the implicit `std`
 /// dependency to the `forc` version.
 pub const SWAY_GIT_TAG: &str = concat!("v", clap::crate_version!());
+
+pub(crate) fn forc_rustc_version() -> Result<Version> {
+    let version = version().map_err(|e| anyhow!("Failed to locate rustc: {}", e))?;
```

Comment:
Sorry I should have clarified - the idea here was to keep the logic where we retrieved the rust version from `forc`'s `Cargo.toml` file, as this is the best indicator of what Rust version will remain compatible with the test harness generated by `forc`. I think we can probably remove the `rustc_version` crate dependency altogether.